citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -99,6 +100,77 @@
         }
     });
 });
+
+$(function() {
+	$(".delete-link").click(function(event) {
+		var ids = []
+		$('.entry-checkbox:checked').each(function() 
+			{
+			   ids.push($(this).data('entry-key'));
+			});
+		if(ids.length==0) {
+			$('#no-selected-entry').modal("show");	
+		}
+		else {
+			$("#delete-entry-button").attr("data-entry-key", ids);
+			$('#entry-delete-confirmation').modal("show");
+		}
+		event.preventDefault();
+	});
+	
+	$("#delete-entry-button").click(function() {
+		var ids = $(this).data("entry-key");
+		var entry = "Entry";
+		var rows = ids.split(",");
+		var baseurl = "references/delete/";
+
+		$.ajax({
```

Comment:
how long does the deletion take? If it takes too long, Apache might time out.

Reply:
This comment is yet to be resolved. I was reading about how to handle ajax calls that take long time. I re-opened PR for your 
review on other changes corresponding to the changes in Spring-social-zotero.

Reply:
instead of making one call and waiting, there should be an async call to start the delete process and then polling to get the results.